### PR TITLE
doc updates for s3, domo, databricks sql, and hex

### PR DESCRIPTION
**Pull request by wrp801**

### CODE REVIEW

File: docs/blueprint-library/amazon-s3/amazon-s3-move-or-rename-files.md [hunk 1]
```diff
@@ -54,7 +60,16 @@ guardrails:
   retry_wait: 0h0m0s
   runtime_cutoff: 1h0m0s
   exclude_exit_code_ranges:
-    - '201'
+    - '202'
     - '203'
+    - '101'
```
Comment (johnathan-rodriguez) [inline docs/blueprint-library/amazon-s3/amazon-s3-move-or-rename-files.md:65]: Small detail but can we fix the order of this

File: docs/blueprint-library/amazon-s3/amazon-s3-remove-files.md [hunk 2]
```diff
@@ -40,16 +46,25 @@ source:
...
   type: BLUEPRINT
 guardrails:
   retry_count: 1
   retry_wait: 0h0m0s
   runtime_cutoff: 1h0m0s
   exclude_exit_code_ranges:
-    - '201'
+    - '202'
     - '203'
+    - '101'
```
Comment (johnathan-rodriguez) [inline docs/blueprint-library/amazon-s3/amazon-s3-remove-files.md:60]: Same here

**Review by johnathan-rodriguez — CHANGES_REQUESTED** [review]
Fix order of exit codes

**Comment by wrp801** [comment]
@johnathan-rodriguez ready for re-review

**Review by johnathan-rodriguez — APPROVED** [review]

**Pull request merged**